Prolangle/Program.cs

```diff
@@ -16,7 +16,20 @@
 {
 	var lp = sp.GetRequiredService<LanguagesProvider>();
 	var logger = sp.GetRequiredService<ILogger<GuessGame>>();
-	return new(lp, logger, () => DateTime.Now.Minute);
+
+	var environment = sp.GetRequiredService<IWebAssemblyHostEnvironment>();
+
+	Console.WriteLine($"Running in {environment.Environment} environment");
```

Comment:
I'd rather not use `Console` directly. Instead, I suggest using an `ILogger`

Reply:
Huh, I thought logging wasn't configured.